articles/creating-your-first-google-web-story/index.md

```diff
@@ -0,0 +1,276 @@
+# Creating your first Google Web Story
+
+![Hero image](engineering-education/creating-your-first-google-web-story/hero.png)
+[Image Source: Unsplash](https://unsplash.com/photos/OU19pS58YY8)
```

Comment:
Could you push this image downwards after the introduction.

Reply:
Alright, Done.